plugins/woocommerce/client/admin/package.json

```diff
@@ -221,6 +221,7 @@
 		"webpack-cli": "5.1.x",
 		"webpack-dev-server": "4.15.x",
 		"webpack-fix-style-only-entries": "^0.6.1",
+		"whatwg-fetch": "^3.6.20",
```

Comment:
This is necessary because removing the `@automattic/data-stores` dependency removed the `whatwg-fetch` dependency, and we are still using that in a test:

https://github.com/woocommerce/woocommerce/blob/a16f11ad6ee01389bdb68ee499b85c676e283d36/plugins/woocommerce/client/admin/client/marketing/hooks/test/useCampaigns.test.ts#L7